ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -70,6 +77,19 @@ class AlertsView extends React.Component {
       this.setState({ totalPages: this.generatePages(count) });
     }
 
+    if (
+      validated.hideImprovements !== prevProps.validated.hideImprovements ||
```

Comment:
I would give a name to `validated.hideImprovements !== prevProps.validated.hideImprovements`, such as `hideImprovementsChanged`.
The same for `hideDwnToInv`.